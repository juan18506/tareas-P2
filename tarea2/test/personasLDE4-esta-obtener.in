crearListaPersonas

crearAgenda
crearPersona 1 11 Persona1
crearFecha 1/11/11
crearEvento 1 Descripcion1
agregarEventoPersona
insertarPersonaListaPersonas 1

crearAgenda
crearPersona 2 22 Persona2
crearFecha 2/22/22
crearEvento 2 Descripcion2
agregarEventoPersona
insertarPersonaListaPersonas 2

crearAgenda
crearPersona 3 33 Persona3
crearFecha 3/33/33
crearEvento 3 Descripcion3
agregarEventoPersona
insertarPersonaListaPersonas 3

estaEnListaPersonas 3
estaEnListaPersonas 5
estaEnListaPersonas 2
estaEnListaPersonas 0

imprimirPersonaListaPersonas 3
imprimirPersonaListaPersonas 1

liberarListaPersonas

Fin