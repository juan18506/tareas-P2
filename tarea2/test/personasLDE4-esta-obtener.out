1>2>3>4>5>6>7>8>9>10>11>12>13>14>15>16>17>18>19>20>La persona con id 3 está en la lista.
21>La persona con id 5 NO está en la lista.
22>La persona con id 2 está en la lista.
23>La persona con id 0 NO está en la lista.
24>Persona 3:  Persona3, 33 años
Evento 3:  Descripcion3
Fecha: 3/33/33
25>Persona 1:  Persona1, 11 años
Evento 1:  Descripcion1
Fecha: 1/11/11
26>27>Fin.
